declarations.d.ts

```diff
@@ -29,6 +29,66 @@ declare module "chrome-trace-event" {
 	}
 }
 
+// There are no typings for @webassemblyjs/ast
+declare module "@webassemblyjs/ast" {
```

Comment:
Could I move these declaration into the `@webassembly` namespace?

Reply:
Currently these are the bare minimum types required for compile successful. If you want to move this to your monorepo or DT you probably need full typings.